__tests__/app.test.js

```diff
@@ -90,6 +90,46 @@ describe("/api/articles", () => {
         });
       });
   });
+
+  test("Get: 200 sends an array of articles queried by the specified topic ", () => {
+    const returnedArticles = [
+      {
+        author: "rogersop",
+        title: "UNCOVERED: catspiracy to bring down democracy",
+        article_id: 5,
+        topic: "cats",
+        created_at: "2020-08-03T13:14:00.000Z",
+        votes: 0,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        comment_count: "2",
+      },
+    ];
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles).toEqual(returnedArticles);
+      });
+  });
```

Comment:
It would also be good to check an example endpoint where there are multiple responses for that topic e.g. `topic=mitch `

Reply:
Sorted. well done 👍